test/functional/p2p_orphan_handling.py

```
@@ -0,0 +1,414 @@
+        # Relay the child. It should not be accepted because it has missing inputs.
+        self.relay_transaction(peer2, child_invalid_witness["tx"])
+        assert child_invalid_witness["txid"] not in node.getrawmempool()
+
+        # The parent should be requested since the unstripped wtxid would differ. Delayed because
+        # it's by txid and this is not a preferred relay peer.
+        self.nodes[0].bumpmocktime(NONPREF_PEER_TX_DELAY + TXID_RELAY_DELAY)
+        peer2.wait_for_getdata([int(parent_normal["tx"].rehash(), 16)])
+
+        # parent_normal can be relayed again even though parent1_witness_stripped was rejected
+        self.relay_transaction(peer1, parent_normal["tx"])
+        assert_equal(set(node.getrawmempool()), set([parent_normal["txid"], child_invalid_witness["txid"]]))
```

Ruff (0.12.2)

Unnecessary list literal (rewrite as a set literal)

Rewrite as a set literal

(C405)